client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -44,12 +44,13 @@ const CustomerHomeLaunchpad = ( {
 		} );
 
 		// Record views for each task.
-		checklist?.map( ( task: Task ) => {
+		checklist?.map( ( task: Task, index: number ) => {
 			recordTracksEvent( 'calypso_launchpad_task_view', {
 				checklist_slug: checklistSlug,
 				task_id: task.id,
 				is_completed: task.completed,
 				context: 'customer-home',
+				order: index,
```

Comment:
I am not sure I like these two separate pieces of code determining the `order` value separately. Might it be worth embedding the positional information in the `task` object after we've performed sorting so we can be confident that we have the same number used in both cases? We could add that to the final object we build in `setUpActionsForTasks()`.

Reply:
There's also another problem that I noticed. The order will vary when the tasks are completed:
https://github.com/Automattic/wp-calypso/blob/e18a187ab583861805c9ac99d9c4b9ae774bbd60/packages/launchpad/src/setup-actions.ts#L25-L27
We use the `sortedTasks` variable to define the order.